tests/test_replace_url/go.mod

```
@@ -0,0 +1,7 @@
+module testmodule
+
+go 1.11
+
+require golang.org/x/text v0.3.2
```

This should fix the test to check that the replaced version does not get used.

```suggestion
require golang.org/x/text v0.0.0-00010101000000-000000000000
```